UTs fails within env folder

/kind flake

<!--

Welcome! - We kindly ask you to:

  1. Fill out the issue template below 
  2. Use the Google group if you have a question rather than a bug or feature request.

The group is at: https://groups.google.com/forum/#!forum/odo-users

Thanks for understanding, and for contributing to the project!

-->

## What versions of software are you using?

**Operating System:**
Windows, Linux

**Output of `odo version`:**
master

## How did you run odo exactly?
make test

## Actual behavior
Observed in travis CI
```
?   	github.com/openshift/odo/pkg/odo/cli/component/ui	[no test files]
?   	github.com/openshift/odo/pkg/odo/cli/config	[no test files]
?   	github.com/openshift/odo/pkg/odo/cli/debug	[no test files]
--- FAIL: TestPrintSupportedParameters (0.00s)
    --- FAIL: TestPrintSupportedParameters/Case_1:_Test_print_supported_set_parameters (0.00s)
        env_test.go:49: Got Available parameters:
              Namespace: Use this value to set component namespace
              DebugPort: Use this value to set component debug port
              Name: Use this value to set component name, want Available parameters:
              Name: Use this value to set component name
              Namespace: Use this value to set component namespace
              DebugPort: Use this value to set component debug port
FAIL
FAIL	github.com/openshift/odo/pkg/odo/cli/env	0.333s
?   	github.com/openshift/odo/pkg/odo/cli/login	[no test files]
?   	github.com/openshift/odo/pkg/odo/cli/logout	[no test files]
?   	github.com/openshift/odo/pkg/odo/cli/plugins	[no test files]
```
## Expected behavior
No such flake should appear

## Any logs, error output, etc?

@amitkrout I can see this flake on linux too

```
?   	github.com/openshift/odo/pkg/odo/cli/debug	[no test files]
--- FAIL: TestPrintSupportedParameters (0.00s)
    --- FAIL: TestPrintSupportedParameters/Case_1:_Test_print_supported_set_parameters (0.00s)
        env_test.go:49: Got Available parameters:
              DebugPort: Use this value to set component debug port
              Name: Use this value to set component name
              Namespace: Use this value to set component namespace, want Available parameters:
              Name: Use this value to set component name
              Namespace: Use this value to set component namespace
              DebugPort: Use this value to set component debug port
FAIL
coverage: 7.2% of statements
FAIL	github.com/openshift/odo/pkg/odo/cli/env	0.067s
FAIL
Makefile:115: recipe for target 'test-coverage' failed
make: *** [test-coverage] Error 1
The command "make test-coverage" exited with 2.
Done. Your build exited with 1.
```

--- a/pkg/odo/cli/env/env_test.go
+++ b/pkg/odo/cli/env/env_test.go
@@ -45,8 +45,20 @@ func TestPrintSupportedParameters(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got := strings.TrimSpace(printSupportedParameters(tt.supportedParameters))
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("Got %s, want %s", got, tt.want)
+
+			gotStrings := strings.Split(got, "\n")
+			wantStrings := strings.Split(tt.want, "\n")
+
+			gotStringMap := make(map[string]bool)
+			for _, gotString := range gotStrings {
+				gotStringMap[gotString] = true
+			}
+
+			for _, wantString := range wantStrings {
+				_, found := gotStringMap[wantString]
+				if !found {
+					t.Errorf("String %s not found in output %s", wantString, got)
+				}
 			}
 		})
 	}

--- a/pkg/odo/cli/env/env_test.go
+++ b/pkg/odo/cli/env/env_test.go
@@ -2,6 +2,7 @@ package env
 
 import (
 	"reflect"
+	"sort"
 	"strings"
 	"testing"
 )
@@ -49,16 +50,11 @@ func TestPrintSupportedParameters(t *testing.T) {
 			gotStrings := strings.Split(got, "\n")
 			wantStrings := strings.Split(tt.want, "\n")
 
-			gotStringMap := make(map[string]bool)
-			for _, gotString := range gotStrings {
-				gotStringMap[gotString] = true
-			}
+			sort.Strings(gotStrings)
+			sort.Strings(wantStrings)
 
-			for _, wantString := range wantStrings {
-				_, found := gotStringMap[wantString]
-				if !found {
-					t.Errorf("String %s not found in output %s", wantString, got)
-				}
+			if !reflect.DeepEqual(wantStrings, gotStrings) {
+				t.Errorf("\nGot: %s\nWant: %s", got, tt.want)
 			}
 		})
 	}

